gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -2246,4 +2345,31 @@ private void checkGradleVersion(Project project)
           MIN_SUGGESTED_VERSION));
     }
   }
+
+  /**
+   * Reflection is necessary to obscure types introduced in Gradle 5.3
```

Comment:
_General comment for the whole PR:_ Make sure to update the Changelog with whatever is relevant to consumers (e.g. compatibility concerns, new behavior).

Reply:
Will do.